src/CoreWCF.Http/tests/SimpleWSHTTPTest.cs

```diff
@@ -195,6 +229,18 @@ public override void ChangeHostBehavior(ServiceHostBase host)
             }
         }
 
+        internal class WSHttpTransportSecurityWindowsAuth : StartupWSHttpBase
+        {
+            public WSHttpTransportSecurityWindowsAuth() : base(CoreWCF.SecurityMode.TransportWithMessageCredential, MessageCredentialType.Windows)
+            {
+            }
+
+            public override void ChangeHostBehavior(ServiceHostBase host)
+            {
```

Comment:
Maybe make the base method virtual instead of abstract then you can remove the do nothing overrides.